examples/adapters/modeling.py

```diff
@@ -0,0 +1,166 @@
+import tensorflow as tf
```

Comment:
I think the naming pattern is to add a prefix `TF` before anything that uses tensorflow instead of pytorch? @LysandreJik